src/modules/shared/pages/innovation/actions/wizard-task-new/steps/message-step.component.html

```diff
@@ -0,0 +1,18 @@
+<theme-content-wrapper [status]="pageStatus">
+
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-two-thirds">
+
+      <form [formGroup]="form" (ngSubmit)="onSubmitStep()">
+        <theme-form-textarea controlName="message" label="Describe the information you want the innovator to provide" description="Enter message" [pageUniqueField]="false" lengthLimit="s"></theme-form-textarea>
```

Comment:
I think the length for the message is bigger than `s`!

Reply:
Figma is saying 2000, but I talked to Mariana and she said that number was only an example for design, so I kept the original length that is 500. It's not the message feature, it's the message sent when the user assigns a new task